selfdrive/ui/ui.cc

```diff
@@ -863,44 +856,26 @@ int main(int argc, char* argv[]) {
       should_swap = true;
     }
 
-    if (s->volume_timeout > 0) {
-      s->volume_timeout--;
-    } else {
-      int volume = fmin(MAX_VOLUME, MIN_VOLUME + s->scene.controls_state.getVEgo() / 5);  // up one notch every 5 m/s
-      set_volume(volume);
-      s->volume_timeout = 5 * UI_FREQ;
-    }
+    s->sound.setVolume(fmin(MAX_VOLUME, MIN_VOLUME + s->scene.controls_state.getVEgo() / 5), 5); // up one notch every 5 m/s
 
     // If car is started and controlsState times out, display an alert
     if (s->controls_timeout > 0) {
       s->controls_timeout--;
     } else {
-      if (s->started && s->controls_seen && s->scene.alert_text2 != "Controls Unresponsive") {
+      if (s->started && s->controls_seen && s->sound.currentSound() != AudibleAlert::CHIME_WARNING_REPEAT) {
```

Comment:
Why change this? Other alerts use this sound, so this won't get triggered if the last alert used this sound